### ISSUE TITLE
Notify on Submission/Evidence Failures

### ISSUE THREAD

**Issue by shaunburdick** [department-of-veterans-affairs/va.gov-team#84372]
## Value Statement

**_As a_** Veteran

**_I want to_** receive notifications if there is an issue with my submission

**_So that_** I can address the issues and/or submit via another avenue

---

## Background Context

- If a submission fails, the system does not notify the user of any issue

## Outcome, Success Measure, KPI(S), and Tracking Link

- If a failure occurs, the user is notified via Email with steps to achieve a favorable outcome if needed

## Acceptance Criteria

- [x] Scan table from #84329 for negative statuses
- [x] Send notifications to needed parties with information to resolve the issue
- [x] Mark the record when the notification is sent

## Designs and Build Notes

- Add here

### Enablement team (if needed)
@ member of team

## Out of scope

- Add here

## Open questions

- "Steal" email content from 526 (@shaunburdick )
- Get sign off on email content from VBA and BVA?
  - John Gosnell
  - Andrew Gray

## Tasks

- [x] _Email Template Markdown for form and evidence failures_
- [x] _Everyone has access to VA Notify Project_
- [x] _Update AppealSubmission[Upload] table to indicate when notification is sent and mirror status from SavedClaims_
- [x] _Create the job to send notifications for failed submissions_
- [ ] _Create/Update dashboard to track notifications sent (Show notifications sent vs pending)_
- [x] _Setup a webhook to accept status responses from VA Notify_

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by anniebtran** [comment]
PR incoming today

**Comment by comaurice** [comment]
PR will be updated later today

**Comment by dfong-adh** [comment]
PR ready for review: https://github.com/department-of-veterans-affairs/vets-api/pull/18861

**Comment by kayline** [comment]
Need to push email templates to prod.
Blocked by #95703

**Comment by dfong-adh** [comment]
Settings have been updated, unblocked.

**Comment by dfong-adh** [comment]
Flag will be enabled in production on Monday.
